src/main/java/seedu/weme/model/util/SampleDataUtil.java

```diff
@@ -56,6 +59,48 @@
         return createSampleTemplates(templateFields, userPrefs);
     }
 
+    public static Records getSampleRecords() {
```

Comment:
This doesn't look like it's working for some reason, I'm not getting these records when I first start up the app.